lib/stat_tracker.rb

```diff
@@ -134,9 +134,30 @@ def worst_offense #issue # 12
   end
 
   def highest_scoring_visitor #issue # 13
+      away_team_ids_array = (games[:away_team_id]).uniq.sort
```

Comment:
You might want to do `@games` here rather than `games` so that we are calling on the instance variable not the attr_reader. (same with line 144 and 148)